lib/v2/gov/shanghai/wsjkw/yqtb/index.js

```diff
@@ -0,0 +1,33 @@
+const cheerio = require('cheerio');
+const got = require('@/utils/got');
+const { parseDate } = require('@/utils/parse-date');
+
+module.exports = async (ctx) => {
+    const url = `https://wsjkw.sh.gov.cn/yqtb/index.html`;
+
+    const res = await got.get(url);
+    const $ = cheerio.load(res.data);
+    const list = $('.uli16.nowrapli.list-date  li');
+    ctx.state.data = {
+        title: '疫情通报-上海卫健委',
+        link: url,
+        item:
+            list &&
+            list
+                .map((index, item) => {
+                    item = $(item);
+                    const title = item.find('a').text();
+                    const address = item.find('a').attr('href');
+                    const host = `https://wsjkw.sh.gov.cn`;
+                    const pubDate = parseDate(new Date(item.find('span').text()));
```

Comment:
```suggestion
                    const pubDate = parseDate(item.find('span').text());
```
parseDate() will return a Date object. No need to new Date.
You should also check if the timezone is correct from Vercel's output.